src/main/resources/xslt/preservica2solr.xsl

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xsl:transform xmlns:t="http://www.test.com/"
+               xmlns:m="http://www.loc.gov/mods/v3"
+               xmlns:mets="http://www.loc.gov/METS/"
+               xmlns:tei="http://www.tei-c.org/ns/1.0"
+               xmlns:f="http://www.w3.org/2005/xpath-functions"
+               xmlns:h="http://www.w3.org/1999/xhtml"
+               xmlns:xs="http://www.w3.org/2001/XMLSchema"
+               xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+               xmlns:my="urn:my"
+               xmlns:premis="http://www.loc.gov/premis/v3"
+               xmlns:mix="http://www.loc.gov/mix/v20"
+               xmlns:pbc="http://www.pbcore.org/PBCore/PBCoreNamespace.html"
+               xmlns:xip="http://example.com/"
```

Comment:
The repo https://github.com/preservica/metadata-management indicates that the XIP namespace is "http://www.tessella.com/XIP/v4" (but of course kb.dk might use a lower version than 4).